### PR TITLE
Add Alpine JS

### PR DESCRIPTION
**Pull request by pcbowers**
This pull request adds Alpine JS. It also refactors some of the links slightly to improve development so that any new logos that are added during a fork can be viewed when developing (this assumes a production build is used when deploying).

### CODE REVIEW

**Comment by pcbowers** [comment]
@matschik Feel free to make changes to things. It's as complete as I could make it, and it should have no conflicts (assuming you don't push any further commits)!

File: src/utils/createAlpineREPL.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
+import { getParameters } from 'codesandbox/lib/api/define';
```
Comment (matschik) [inline src/utils/createAlpineREPL.js:1]: Wow I didn't know you can do that with codesandbox.
It looks dope ! 🔥

File: src/utils/createAlpineREPL.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+export default function createVue3REPL() {
```
Comment (matschik) [inline src/utils/createAlpineREPL.js:3]: rename function for Alpine

File: src/components/FrameworkLabel.astro [hunk 3]
```diff
@@ -3,9 +3,11 @@ import FRAMEWORKS from '../frameworks';
 const {id, size = 20} = Astro.props
 
 const framework = FRAMEWORKS.find((f) => f.id === id);
+
+const baseURL = import.meta.env.MODE === 'development' ? '' : 'https://raw.githubusercontent.com/matschik/component-party/main/public/';
```
Comment (matschik) [inline src/components/FrameworkLabel.astro:7]: Wow nice !! You solved the image problem !! 💯

**Review by matschik — COMMENTED** [review]
Amazing so far !! 🤩 
I learned some tricks by reading your PR.
Thanks a lot !

**Comment by matschik** [comment]
Seeing that the "Component composition" section is not possible for Alpine, I don't know what's the best solution to illustrate that.

The first comment is fine at the moment but I think we need to show a stronger indicator.
I created #54 to solve this problem.

File: src/components/FrameworkLabel.astro (hunk 3 above)
Comment (pcbowers) [reply src/components/FrameworkLabel.astro:7]: I'm not nearly as familiar with Astro as you are nor am I familiar with how you deploy, but as long as you use the production mode in deployment I think it should work!

**Comment by pcbowers** [comment]
> Seeing that the "Component composition" section is not possible with Alpine, I don't know what's the best solution to illustrate that.
> 
> The first comment is fine at the moment but I think we need to show a stronger indicator.
> 
> I created #54 to solve this problem 😄

Yeah, that's a tricky one! It's also sort of in-between since you can create data components using Alpine.data() and Alpine.store(), but not HTML components. It also supports custom event dispatches just fine which you would think would require HTML components. An Issue is a good idea!

File: src/utils/createAlpineREPL.js (hunk 1 above)
Comment (pcbowers) [reply src/utils/createAlpineREPL.js:1]: You'll have to let me know if we need to make changes here! I had to add `codesandbox` as a dependency. I installed it as a dev dependency assuming it pre-bundles everything and no JavaScript is actually run once it is deployed to the website, but I don't know if this is the case.

Comment (matschik) [inline src/utils/createAlpineREPL.js:1]: `import` in NodeJS needs to specify the extension: "codesandbox/lib/api/define.js"

**Comment by matschik** [comment]
Regenerate the lock file and I think we are good to go !

**Comment by pcbowers** [comment]
@matschik You may want to make a quick fix for ember (in terms of image url's) now that we've updated the URL's. My original fork did not include changes for ember.

**Comment by matschik** [comment]
Yup ! Just solved it !

**Pull request merged**